tfjs-converter/python/tensorflowjs/converters/keras_tfjs_loader.py

```diff
@@ -108,6 +170,30 @@ def _get_weights_manifest_from_config_json(config_json):
         'Field "weightsManifest" is missing from the JSON content.')
   return config_json['weightsManifest']
 
+def _generate_v3_keys(config):
+  if isinstance(config, dict):
+    list_of_keys = list(config.keys())
+    for key in list_of_keys:
+      _generate_v3_keys(config[key])
+    if 'class_name' in list_of_keys:
+      config['module'] = _get_module(config['class_name'])
+      # Put registred name as None since we do not support
+      # custom object saving when we save the model.
+      config['registered_name'] = None
+
+  elif isinstance(config, list):
+    for item in config:
+      _generate_v3_keys(item)
+
+def _generate_meta_json(version_number):
+  meta_data = {}
+  meta_data['keras_version'] = version_number
+  meta_data['date_saved'] = datetime.datetime.now().strftime("%Y-%m-%d@%H:%M:%S")
+
+def _get_module(class_name):
+  module_path = tf_module_mapper.get_module_path(class_name)
+  return module_path
```

Comment:
This seems like a really simple function. Should we just call `tf_module_mapper.get_module_path` instead?

Reply:
SG! Removed.